julia/http/Project.toml

```
@@ -0,0 +1,5 @@
+[deps]
+HTTP = "cd3eb016-35fb-5094-929b-558a96fad6f3"
+
+[compat]
+HTTP = "1"
```

```suggestion
HTTP = "1.10"
```

In order to stick to 1.10, version will bé displayed as X.Y on results